src/components/FiltersDrawer/index.tsx

```diff
@@ -141,18 +144,58 @@ const FiltersDrawer: React.FC<FiltersDrawerProps> = ({
                 <Overline>
                   Row {e[0]} to {e[1]}
                 </Overline>
-              </ExpandedLineWrapper>
+              </NonFilterElement>
             ))
           ) : (
-            <ExpandedLineWrapper data-cy="no-expanded-lines-message">
+            <SectionWrapper data-cy="no-expanded-lines-message">
               <Body>No lines have been expanded.</Body>
-            </ExpandedLineWrapper>
+            </SectionWrapper>
+          )}
+        </StyledSideNavGroup>
+
+        <StyledSideNavGroup
+          glyph={<SideNavIcon active={highlights.length > 0} glyph="Minus" />}
+          header={
+            <NavGroupHeader data-cy="highlight-nav-group-header">
+              <NavGroupTitle>Highlighted Terms</NavGroupTitle>
+              <Badge variant={Variant.Green}>{highlights.length}</Badge>
+            </NavGroupHeader>
+          }
+        >
+          {highlights.length ? (
+            <>
+              {highlights.map((highlight) => (
+                <NonFilterElement key={`highlight-${highlight}`}>
+                  <IconButton
+                    aria-label="Delete highlight"
+                    data-cy="delete-highlight-button"
+                    onClick={() => deleteHighlight(highlight)}
+                  >
+                    <Icon fill={green.dark2} glyph="X" />
+                  </IconButton>
+                  <Overline>{highlight}</Overline>
+                </NonFilterElement>
+              ))}
+            </>
```

Comment:
nit. unnecessary fragment (lint is probably very wrong sry)
```suggestion
highlights.map((highlight) => (
              <NonFilterElement key={`highlight-${highlight}`}>
                <IconButton
                  aria-label="Delete highlight"
                  data-cy="delete-highlight-button"
                  onClick={() => deleteHighlight(highlight)}
                >
                  <Icon fill={green.dark2} glyph="X" />
                </IconButton>
                <Overline>{highlight}</Overline>
              </NonFilterElement>
            ))
```